### PR TITLE
Onboard UI Update

### PR DESCRIPTION
**Pull request by Nate-Teall**
Most of what is changed is in Game1.cs, Menu.cs, and MenuCard.cs. Mostly visual changes like animated background and loading screens. The backend stuff with getting games/assets from the server is unchanged and will need to be fixed still.

### CODE REVIEW

**Comment by MTFT-Games** [comment]
suggestion: use sprite maps instead of 70 individual frames

**Comment by Nate-Teall** [comment]
> suggestion: use sprite maps instead of 70 individual frames

Alright, I will do that next

**Comment by WillNilges** [comment]
```/Devcade-onboard/onboard/Content/devcade-menu-big.spritefont : error : Processor 'FontDescriptionProcessor' had unexpected failure! [/Devcade-onboard/onboard/onboard.csproj]
/Devcade-onboard/onboard/Content/devcade-menu-title.spritefont : error : Processor 'FontDescriptionProcessor' had unexpected failure! [/Devcade-onboard/onboard/onboard.csproj]
/root/.nuget/packages/monogame.content.builder.task/3.8.0.1641/build/MonoGame.Content.Builder.Task.targets(138,5): error MSB3073: The command "dotnet /root/.nuget/packages/monogame.content.builder.task/3.8.0.1641/build//../tools/netcoreapp3.1/any/mgcb.dll /quiet /@:"/Devcade-onboard/onboard/Content/Content.mgcb" /platform:DesktopGL /outputDir:"/Devcade-onboard/onboard/Content/bin/DesktopGL/Content" /intermediateDir:"/Devcade-onboard/onboard/Content/obj/DesktopGL/Content" /workingDir:"/Devcade-onboard/onboard/Content/"" exited with code 2. [/Devcade-onboard/onboard/onboard.csproj]

The build failed. Fix the build errors and run again.```

File: onboard/Content/devcade-menu-big.spritefont [hunk 1]
```diff
@@ -10,6 +10,7 @@ with.
 
     <!--
     Modify this string to change the font that will be imported.
+    /usr/share/fonts/truetype/liberation/LiberationMono-Bold.ttf
```
Comment (WillNilges) [inline onboard/Content/devcade-menu-big.spritefont]: Delete this comment?

**Review by WillNilges — CHANGES_REQUESTED** [review]

**Review by WillNilges — APPROVED** [review]
:shipit:

**Pull request merged**